std_msgs/Header header
float64 tx
float64 ty
float64 tz
float64 vx
float64 vy
float64 vz
